### PR TITLE
Templating review

### PR DESCRIPTION
**Pull request by myarjunar**
### What does it fix?
* Ticket: #4303 
* Funded by: DFAT
* Description: Enable custom hazard-exposure specific template to be generated and rename template file to the generic one. 

The naming convention for this hazard-exposure template is based on hazard and exposure definition key. 
selection_135

cc: @timlinux

### CODE REVIEW

**Comment by Gustry** [comment]
@myarjunar You are on fire, you are doing tickets for 4.3 #4378 ;-)

**Comment by myarjunar** [comment]
I thought it was for 4.2 hha. but Tim said that it would be nice if we could have it in 4.2. Anyway, can we merge this? because another PR will be coming after this PR has been merged. @timlinux @Gustry

File: safe/definitions/utilities.py [hunk 1]
```diff
@@ -52,7 +53,7 @@ def purposes_for_layer(layer_geometry_key):
 
 
 def hazards_for_layer(layer_geometry_key, hazard_category_key=None):
-    """Get hazard categories form layer_geometry_key.
+    """Get hazard categoriewalk, s form layer_geometry_key.
```
Comment (ismailsunni) [inline safe/definitions/utilities.py]: typo?

**Comment by timlinux** [comment]
@myarjunar can you make the file naming more consistent by using only underscores or only dashes between words?

**Pull request merged**